lib/llm/response/chunk.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module LLM
+  require_relative "completion"
```

Comment:
Can we move this to `lib/llm/response.rb` instead ? The `require_relative` statements found in that file are properly ordered. For example, it could be like this and the dependencies should be properly met:
```ruby
    require "json"
    require_relative "response/completion"
    require_relative "response/chunk"
    require_relative "response/embedding
```